addon/mixins/picker.js

```diff
@@ -8,17 +9,26 @@ const {
   Mixin,
 } = Ember;
 
-export default Mixin.create({
-  isExpanded: false,
+export default Mixin.create(CancelableMixin, {
+  dateFormat: "MM/DD/YYYY",
+  tabIndex: 1,
+  showClear: true,
   startDate: moment().startOf('date'),
   endDate: moment().startOf('date'),
+  startMonth: moment().startOf('month'),
+  endMonth: moment().startOf('month'),
+
+  dropdownController: Ember.Object.create({
+    isOpen: false,
+  }),
+  initiallyOpened: false,
 
   _startDateToMoment: on('init', observer('startDate', function() {
```

Comment:
Kill all observers. I know you're not responsible for this, so no need to fix now. But, I think we could probably just move what we're doing here to `didReceiveAttrs` as well.